src/game/objects/har.c

```diff
@@ -533,9 +533,15 @@ void har_move(object *obj) {
         obj->pos.y = ARENA_FLOOR;
 
         char last_input = get_last_input(h);
-        // Change animation from jump to walk or idle,
-        // depending on held inputs
-        if(h->state == STATE_JUMPING && enemy_har->is_grabbed == 0) {
+        if(h->state == STATE_VICTORY && obj->vel.y > 0) {
+            object_set_vel(obj, vec2f_create(0, 0));
```

Comment:
Can you comment the intent here, if the HAR is in VICTORY state, but has landed on the floor, set them into the idle animation?